### PR TITLE
Adding adi_tmc_coe to documentation index for distro

### PR DESCRIPTION
**Pull request by CAcarADI**
# Please Add This Package to be indexed in the rosdistro.

adi_tmc_coe

# The source is here:

https://github.com/analogdevicesinc/adi_tmc_coe

# Checks
 - [ ] All packages have a declared license in the package.xml
 - [ ] This repository has a LICENSE file
 - [ ] This package is expected to build on the submitted rosdistro

### CODE REVIEW

File: noetic/distribution.yaml [hunk 1]
```diff
@@ -75,6 +75,16 @@ repositories:
       url: https://github.com/ros/actionlib.git
       version: noetic-devel
     status: maintained
+  adi_tmc_coe:
+    doc:
+      type: git
+      url: https://github.com/analogdevicesinc/adi_tmc_coe
```
Comment (clalancette) [inline noetic/distribution.yaml:81]:
```suggestion
      url: https://github.com/analogdevicesinc/adi_tmc_coe.git
```

**Review by clalancette — CHANGES_REQUESTED** [review]
In addition to the change inline, this also needs to be rebased due to some unrelated changes in this area of the file.

**Comment by CAcarADI** [comment]
Hi @clalancette , Ive already pushed the suggested commit and rebased the commit but the Merging is blocked, is there anything I need to do to fix this?

**Comment by clalancette** [comment]
### New package review checklist

- [x] At least one of the following must be present
  - [x] Top level license file: https://github.com/analogdevicesinc/adi_tmc_coe/blob/noetic/LICENSE
  - [ ] Per package license files:
- [x] License is [OSI-approved](https://opensource.org/licenses): https://opensource.org/license/bsd-3-clause
- [x] License correctly listed in package.xmls
- [x] Public source repo: https://github.com/analogdevicesinc/adi_tmc_coe
- [x] Source repository contains ROS packages
- [x] Each package meets [REP-144](https://www.ros.org/reps/rep-0144.html) naming conventions

**Review by clalancette — APPROVED** [review]

**Pull request merged**